ci-automation/garbage_collect_cloud.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -euo pipefail
+timeout --signal=SIGQUIT 60m ore aws gc
```

Comment:
Could we explicitly pass the credentials ?
```suggestion
timeout --signal=SIGQUIT 60m ore aws gc --access-id "${AWS_ACCESS_KEY_ID}" --secret-key "${AWS_SECRET_ACCESS_KEY}"
```

It helps later to understand who's using what.